certify-default.properties

```diff
@@ -2,6 +2,18 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+### DataProvider plugin specific changes
+# tells Certify to become a proxy to other issuers via VCIssuancePlugin impl
+mosip.certify.issuer=PluginIssuer
+mosip.certify.data-provider.identifier.column=id
+mosip.certify.data-provider.fields.include=id,name,phoneNumber,dateOfBirth,highestEducation,typeOfHouse,numberOfDependents,works,landArea,landOwnershipType,primaryCropType,secondaryCropType,maritalStatus
+mosip.certify.plugin.csv.file.uri=https://raw.githubusercontent.com/jainhitesh9998/digital-credential-plugins/refs/heads/develop/mock-certify-plugin/src/test/resources/farmer_identity_data.csv
+mosip.certify.issuer.vc-sign-algo=Ed25519Signature2020
+mosip.certify.mock.vciplugin.issuer.key-cert="dummy-issuer-cert"
+# this has to be an empty value iff no svg template is configured for VCI templating
+mosip.certify.issuer.svg.template.id=
+mosip.certify.issuer.pub.key=
+mosip.certify.issuer.uri=
```

Comment:
any New properties you are added, should be added to the end and not beginning

Reply:
Moved new properties to the end of the file